README.md

```diff
@@ -57,102 +57,19 @@ _Preview means that this is a work in progress and NNCF does not guarantee the f
-
-# Now use compressed_model as a usual torch.nn.Module 
-# to fine-tune compression parameters along with the model weights
-
-# ... the rest of the usual PyTorch-powered training pipeline
-
-# Export to ONNX or .pth when done fine-tuning
-compression_ctrl.export_model("compressed_model.onnx")
-torch.save(compressed_model.state_dict(), "compressed_model.pth")
-```
-
-**NOTE (PyTorch)**: Due to the way NNCF works within the PyTorch backend, `import nncf` must be done before any other import of `torch` in your package _or_ in third-party packages that your code utilizes, otherwise the compression may be applied incompletely.
-
-
-### Usage example with TensorFlow
-```python
-import tensorflow as tf
-
-from nncf import NNCFConfig
-from nncf.tensorflow import create_compressed_model, register_default_init_args
-
-# Instantiate your uncompressed model
-from tensorflow.keras.applications import ResNet50
-model = ResNet50()
-
-# Load a configuration file to specify compression
-nncf_config = NNCFConfig.from_json("resnet50_int8.json")
-
-# Provide dataset for compression algorithm initialization
-representative_dataset = tf.data.Dataset.list_files("/path/*.jpeg")
-nncf_config = register_default_init_args(nncf_config, representative_dataset, batch_size=1)
-
-# Apply the specified compression algorithms to the model
-compression_ctrl, compressed_model = create_compressed_model(model, nncf_config)
-
-# Now use compressed_model as a usual Keras model
-# to fine-tune compression parameters along with the model weights
-
-# ... the rest of the usual TensorFlow-powered training pipeline
-
-# Export to Frozen Graph, TensorFlow SavedModel or .h5  when done fine-tuning 
-compression_ctrl.export_model("compressed_model.pb", save_format='frozen_graph')
-```
-
-For a more detailed description of NNCF usage in your training code, see [this tutorial](docs/Usage.md). 
-For in-depth examples of NNCF integration, browse the [sample scripts](#compression-aware-training-samples) code, or the [example patches](#third-party-repository-integration) to third-party repositories.
-For FAQ, visit this [link](./docs/FAQ.md).
-
-### Usage examples of Post-Training Quantization
-
-NNCF provides [samples](#post-training-quantization-samples), which demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.
-
-To start the algorithm, provide the following entities:
-* Original model.
-* Validation part of the dataset.
-* [Data transformation function](./docs/compression_algorithms/post_training/Quantization.md#data-transformation-function) transforming data items from the original dataset to the model input data. 
-
-
-The basic workflow steps:
-1) Create the [data transformation function](./docs/compression_algorithms/post_training/Quantization.md#data-transformation-function).
+To run the algorithm you only need your model and the validation part of the dataset without labels. The basic workflow steps:
```

Comment:
What do you think about reusing the introduction from the https://docs.openvino.ai/latest/basic_qauntization_flow.html#doxid-basic-qauntization-flow

Reply:
`The basic quantization flow is the simplest way to apply 8-bit quantization to the model.` -- I agree, added in a slightly modified way

`- It is available for models in the following frameworks: PyTorch, TensorFlow 2.x, ONNX, and OpenVINO.` -- this is obvious from the code blocks lower in the README

`- Set up an environment and install dependencies.` -- too obvious IMO

`- Prepare the calibration dataset that is used to estimate quantization parameters of the activations within the model.` -- (1) too abstract compared to the one that is present (2) I don't think we need to explain to a user what it is used for at this stage

`- Call the quantization API to apply 8-bit quantization to the model.` -- not meaningfully different from the current version

Reply:
Could you reformulate this sentence `To run the algorithm you only need your model and the validation part of the dataset without labels.` because the user doesn't need `the validation part of the dataset without labels.` but needs calibration dataset. Usually, the calibration dataset is a small, no label, representative dataset (~100-500 samples) that is used to estimate the range, i.e. (min, max) of all floating point activation tensors in the model, to initialize the quantization
parameters. The easiest way to define a calibration dataset is to use a training or validation dataset and a transformation function to remove labels from the data item and prepare model input data.

Reply:
Thanks for the detailed information. I reformulated the dataset description, but omitting details. Please take a look.

Regarding the detailed description of calibration dataset, we could add it to a separate more detailed PTQ Usage description documentation file, when we decide to create such in the future. The current `Usage.md` is for NNCF QAT only.